engine/runtime-parser/src/main/java/org/enso/compiler/core/ir/type/Set.java

```diff
@@ -0,0 +1,221 @@
+     * @param value the member's value, if given
+     * @param identifiedLocation the source location that the node corresponds to
+     * @param passData the pass metadata associated with this node
+     */
+    @GenerateFields
+    public Member(
+        @IRChild Name label,
+        @IRChild Expression memberType,
+        @IRChild Expression value,
+        IdentifiedLocation identifiedLocation,
+        MetadataStorage passData) {
+      super(label, memberType, value, identifiedLocation, passData);
+    }
+
+    public static Builder builder() {
+      return new Builder();
+    }
+
+    public Builder copyBuilder() {
+      return new Builder(this);
+    }
+
+    @Override
+    public String showCode(int indent) {
+      var typeString = " : " + memberType().showCode(indent);
+      var valueString = " = " + value().showCode(indent);
+      return "(" + label().showCode(indent) + typeString + valueString + ")";
+    }
+  }
+
+  /** The typeset subsumption judgement {@code <:}. */
+  @GenerateIR(interfaces = {Set.class, IRKind.Primitive.class})
+  final class Subsumption extends SetSubsumptionGen {
+    /**
+     * @param left the left operand
+     * @param right the right operand
+     * @param identifiedLocation the source location that the node corresponds to
+     * @param passData the pass metadata associated with this node
+     */
+    @GenerateFields
+    public Subsumption(
+        @IRChild Expression left,
+        @IRChild Expression right,
+        IdentifiedLocation identifiedLocation,
+        MetadataStorage passData) {
+      super(left, right, identifiedLocation, passData);
+    }
+
+    public static Builder builder() {
+      return new Builder();
+    }
+
+    public static String name() {
+      return "<:";
+    }
+
+    public Builder copyBuilder() {
+      return new Builder(this);
+    }
+
+    @Override
+    public String showCode(int indent) {
+      return "(" + left().showCode(indent) + " <: " + right().showCode(indent) + ")";
+    }
+  }
+
+  /** The typeset equality judgement {@code ~}. */
+  @GenerateIR(interfaces = {Set.class, IRKind.Primitive.class})
+  final class Equality extends SetEqualityGen {
+    @GenerateFields
+    public Equality(
+        @IRChild Expression left,
+        @IRChild Expression right,
+        IdentifiedLocation identifiedLocation,
+        MetadataStorage passData) {
+      super(left, right, identifiedLocation, passData);
+    }
+
+    public static Builder builder() {
+      return new Builder();
+    }
+
+    public static String name() {
+      return "~";
+    }
+
+    public Builder copyBuilder() {
+      return new Builder(this);
+    }
+
+    @Override
+    public String showCode(int indent) {
+      return "(" + left().showCode(indent) + " ~ " + right().showCode(indent) + ")";
+    }
+  }
+
+  /** The typeset concatenation operator {@code ,}. */
+  @GenerateIR(interfaces = {Set.class, IRKind.Primitive.class})
+  final class Concat extends SetConcatGen {
```

Comment:
- I don't think we use `Concat` anywhere
- and I have to admit, I don't know what _concat_ is supposed to mean in a (type) set theory!